packages/ember-test-app/app/templates/index.hbs

```diff
@@ -1,54 +1,63 @@
-<Header class="mb-4">
-  <:center>
-    <p class="p-2 m-0 fs-4 fw-bold text-nowrap">Welcome to Ember Test App</p>
-  </:center>
-</Header>
-<div class="px-3">
-  <div class="my-3">
-    <LinkTo @route="marketing" class="btn btn-outline-dark" role="link">
-      Marketing Test
-      <i class="text-success bi-currency-dollar"></i>
-    </LinkTo>
-
-    <LinkTo @route="form" class="btn btn-outline-primary" role="link">
-      Form Test
-      <i class="text-danger bi-input-cursor-text"></i>
-    </LinkTo>
-
-    <LinkTo @route="faq" class="btn btn-outline-success" role="link">
-      FAQs Test
-      <i class="bi-plus"></i>
-    </LinkTo>
+<main class="container-fluid">
```

Comment:
This container-fluid class fixes the issue of extraneous width which caused a horizontal scroll bar.